### DISCUSSION TITLE
Day specific alerts

### DISCUSSION THREAD

**Discussion by khskekec** [NightscoutFoundation/xDrip#3636] [Ideas]
Hi everyone,

I am pretty sure that my feature idea was already proposed but I was not able to find it here:

Are there any reasons why one can not define weekdays a alarm should be applied on...

This would be great to allow different alarms for working/school days and a special definition for weekends.

Kind regards 

Selcuk

**Comment by Navid200** [comment]
I am not the owner of this repository.  This is my opinion.

I suppose the intent is to have an alert that is louder for when you go to work and a quieter alert for when you are at home?

Or let's say someone is a bus driver and while at work, she prefers to have a slightly higher target glucose in order to guarantee will never accidentally jeopardize passengers safety.

I have two concerns.
My basic concern is that there are already too many features and settings in xDrip.  All of them need to be maintained.  I would like us to limit the features we add to those that are absolutely necessary as opposed to those that may be only nice to have.

My concern specific to this is what happens when you decide to go to work on a weekend or if you decide to stay home on a weekday?
And don't say that it is the responsibility of the user to use this or not so we don't have to worry about how this may go wrong.

With respect to how loud or quiet the alarm is, xDrip already has an ascending volume profile.  You can use it to hopefully address the need for being quiet or loud dynamically.
https://navid200.github.io/xDrip/docs/Ascending-volume-profile.html

Sometimes we find a way to address a need differently than originally suggested.  Can you explain exactly how this can be life-changing?

**Reply by khskekec** [reply]
Hey,

Thank you for your fast response. You got my point and described the usecase very well.

Your arguments against having such a feature by excluding responsibility are getting very weak but okay...

In my opinion a day option would just complement the time condition of the alarms. Using day option requires the same responsibility like using time conditions, it is already there. There is no difference.

Nevertheless i am thankful for getting some feedback and am interested in if such ideas were suggested in the past and reasons against implementing it...

Ahh forgot you're argument about maintainability, sry... This is definitely an issue for sure.

Kind regards 

Selcuk
